CHANGELOG.md

```diff
@@ -57,15 +57,14 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   - Open MPI: `openmpi40-aws-4.1.5-4`
 - Upgrade Slurm to version 23.02.4.
 - Upgrade ARM PL to version 23.04.1 for Ubuntu 22.04 only.
-- Upgrade Cinc Client to version 18.2.7.
```

Comment:
It looks like you are right in that Cinc was not upgraded with 3.7.0. Was this a mistake in the writing of the changelog?

Reply:
I think we didnt change the changelog for develop branch.